app/src/main/res/values-vi/strings.xml

```diff
@@ -0,0 +1,205 @@
+    <string name="description_is_empty">Mô tả không được để trống</string>
+    <string name="create_shop_successful">Tạo cửa hàng thành công!</string>
+    <string name="please_wait">Làm ơn chờ..</string>
+    <string name="loading">Đang tải…</string>
+    <string name="you_must_choose_image">Bạn cần phải chọn ảnh!</string>
+    <string name="blocked">Đã bị chặn</string>
+    <string name="wait">Chờ</string>
+    <string name="active">active</string>
+    <string name="inactive">inactive</string>
+    <string name="create_new_product">Tạo sản phẩm</string>
+    <string name="product_name">Tên sản phẩm</string>
+    <string name="category">Thể loại</string>
+    <string name="start_hour">Giờ bắt đầu</string>
+    <string name="end_hour">Giờ kết thúc</string>
+    <string name="price">Giá</string>
+    <string name="update_success">CẬp nhật thành công</string>
+    <string name="top_shop">Cửa hàng nổi bật</string>
+    <string name="categories">Thể loại</string>
+    <string name="top_product">Sản phẩm nổi bật</string>
+    <string name="create_product_successful">Tạo sản phẩm thành công!</string>
+    <string name="edit_product">Chỉnh sửa sản phẩm</string>
+    <string name="update_successful">Cập nhật thành công</string>
+    <string name="note">Note :</string>
+    <string name="order_successful">Đặt hàng thành công!</string>
+    <string name="accepted_status">accepted</string>
+    <string name="rejected_status">rejected</string>
+    <string name="input_your_note">Nhập ghi chú</string>
+    <string name="add_note">Thêm ghi chú</string>
+    <string name="reject_all">Từ chối tất cả</string>
+    <string name="accept_all">Chấp nhận tất cả</string>
+    <string name="leave">Rời khỏi</string>
+    <string name="close_order">Đóng đơn hàng</string>
+    <string name="close_order_success">Đóng đơn hàng thành công</string>
+    <string name="done_orders">Kết thúc đơn hàng</string>
+    <string name="rejected_orders">Từ chối đơn hàng</string>
+    <string name="name_domain">Tên miền</string>
+    <string name="secret">secret</string>
+    <string name="professed">professed</string>
+    <string name="add_domain">Thêm miền</string>
+    <string name="add_domain_success">Thêm miền thành công!</string>
+    <string name="quantity_order_history">Số lượng: </string>
+    <string name="msg_delete_domain">Bạn có thật sự muốn xóa miền này?</string>
+    <string name="msg_leave_domain">Bạn thực sự muốn rời miền?</string>
+    <string name="delete_the_domain_success">Xóa miền thành công!</string>
+    <string name="leave_the_domain_successful">Rời khỏi miền thành công!</string>
+    <string name="edit_domain">Chỉnh sửa miền</string>
+    <string name="add_to_cart_success">Thêm vào giỏ hàng thành công!</string>
+    <string name="add_to_cart">Thêm vào giỏ hàng</string>
+    <string name="order_now">Đặt ngay</string>
+    <string name="hint_notes">Bất kỳ ý kiến như ít dừa, ít ngọt có thể được đề xuất ở đây...</string>
```

Comment:
WARNING: [TypographyEllipsis] Replace "..." with ellipsis character (…, &#8230;) ?
 You can replace the string "..." with a dedicated ellipsis character, ellipsis character (…, &#8230;). This can help make the text more readable.